src/Main.elm

```diff
@@ -46,6 +51,17 @@ init =
         watchDate movie =
             Movie.watchDate movie
                 |> Maybe.withDefault (date 1800 1 1)
+
+        genres =
+            MovieList.movies
+                |> List.map .genres
+                |> List.map (Set.map String.toLower)
```

Comment:
The part in parens `Set.map String.toLower` is curries. So It returns a function which takes a `Set` and applies `String.toLower` to each item in the set. The outer part takes every item in the List from the previous line, and runs it through this map.

Reply:
I wonder if it would be simpler to do this as part of generating the movie data, so we wouldn't have to do a `.toLower` every time we want to compare on genres. We  could even go as far as making them a union type, so there's no possiblity of casing/typo issues.